integration_tests/integration/serviceProviderReferrals.spec.js

```diff
@@ -25,11 +25,13 @@ describe('Service provider referrals dashboard', () => {
 
     const personalWellbeingIntervention = interventionFactory.build({
       contractType: { code: 'PWB', name: 'Personal wellbeing' },
+      title: 'personal wellbeing - west midlands',
```

Comment:
Is this what the data actually looks like in the interventions service? Seems odd that they're writing "west midlands" in lower case in the first place…

Reply:
I'm not sure to be honest, this was more to test that the uppercasing works.